src/solvers/fc2d_geoclaw/fc2d_geoclaw_fort.h

```diff
@@ -510,11 +510,11 @@ void FC2D_GEOCLAW_FORT_CONSERVATION_CHECK(const int *mx,
                       FCLAW_F77_FUNC(fc2d_geoclaw_fort_compute_patch_area, \
                       FC2D_GEOCLAW_FORT_COMPUTE_PATCH_AREA)
 
-double FC2D_GEOCLAW_FORT_COMPUTE_PATCH_AREA(const int *mx, 
-                                            const int *my, 
-                                            const int *mbc, 
```

Comment:
Here, we just removed the 'const` although eventually, these should all be const.